src/main/java/supersymmetry/common/blocks/BlockResource1.java

```diff
@@ -20,7 +20,9 @@ public BlockResource1() {
     }
 
     public enum ResourceBlockType implements IStringSerializable {
-        NATIVE_COPPER("native_copper", 1);
+        NATIVE_COPPER("native_copper", 1),
+        ANTHRACITE("anthracite", 0),
+        LIGNITE("lignite", 0);
```

Comment:
these lines should be moved to src/main/java/supersymmetry/common/blocks/BlockResource.java, since coal deposit block is in that file.

Reply:
we've used up meta values in `BlockResource` so we can't add more variants to it (that's also wht BlockResource1 exists)

Reply:
I guess the reason is blocks can only have maximum 16 meta values?
Why is GT machines exempted tho, since gregtech:machine meta values can be thousands?

Reply:
BlockMachine actually have just 2 states (2 metas), they hacked it by using damage which is `short`